modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1325,6 +1327,30 @@ export class Wallet implements IWallet {
       }
     }
 
+    // Validate EVM keyring params, referenceAddress is required and referenceCoin is optional for EVM keyring
```

Comment:
if `referenceCoin` by itself isn't valid, it should't be an independent option. Let's instead have `referenceAddressEvmKeyRing` and have the address and coin as nested fields.